src/remediations/controller.fifi.js

```diff
@@ -161,22 +176,41 @@ exports.getSystems = errors.async(async function (req, res) {
         }
     }
 
-    const formatted = format.playbookSystems(systems, count);
+    // Pagination
+    const total = fifi.getListSize(systems);
+    systems = await fifi.pagination(systems, total, limit, offset);
+
+    if (offset >= Math.max(total, 1)) {
```

Comment:
might be better to do the check first before using the value in a call to `fifi.pagination()`